docs/run/advanced-runtime-options.mdx

```diff
@@ -1,25 +1,68 @@
 ---
-title: Algorithm tuning options
-description: Common settings to use when tuning primitive options
+title: Advanced runtime options
+description: Specify options when building with primitives 
 
 ---
 
-<span id="tuning-opt"></span>
-# Algorithm tuning options
+# Advanced runtime options
 
+When calling the primitives, you can pass in options by using the `Options` class or when using the `run` method. In the `Options` class, commonly used options, such as `resilience_level`, are at the first level. Other options are grouped into different categories:
 
-One of the advantages of the primitives is that they abstract away the circuit execution setup so that algorithm developers
-can focus on the pure algorithmic components. However, sometimes, to get the most out of an algorithm, you might want
-to tune certain primitive options. This section describes some of the common settings you might need.
+![options](/images/build/options.png)
 
 <Admonition type="info" title="Attention">
-
-    This section focuses on Qiskit Runtime primitive `Options` (imported from ``qiskit_ibm_runtime``). While
-    most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
+    This section focuses on Qiskit Runtime primitive [Options](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.Options) (imported from ``qiskit_ibm_runtime``). While most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
```

Comment:
The double \` is a holdover from RST. It should be a single back tick. I know this was already like this, but it'd be good to fix here. There are a couple throughout the file.